okio/jvm/jmh/src/jmh/java/com/squareup/okio/benchmarks/IndexOfElementBenchmark.java

```diff
@@ -40,7 +40,7 @@
 @BenchmarkMode(Mode.SampleTime)
 @OutputTimeUnit(TimeUnit.MICROSECONDS)
 public class IndexOfElementBenchmark {
-  ByteString byteString = ByteString.encodeUtf8("abcd");
+  final ByteString byteString = ByteString.encodeUtf8("abcd");
```

Comment:
I don’t think `final` on locals in Java is a good tradeoff for legibility vs. safety

Reply:
Reverted.